### DISCUSSION TITLE
Change Size of the Chatbot Button

### DISCUSSION THREAD

**Discussion by cpapxl** [FlowiseAI/Flowise#781] [Q&A] [answered]
Hello community,

I would like to change the size of the chatbot button embedded on my website.
The default size is set to "medium":
<img width="571" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/142223593/fb9e8a72-43ac-4673-89c0-6f06b1d66921">

I can change it to "large". But there no other possibilities. Is it possible to make it larger? By specifying absolute lengths or relative lengths?

Thanks.

**Comment by SmartMart69** [comment]
I would also be very interested in this!

**Reply by simplecloudbox** [reply]
You can enter size in px like this:

**size: 80,**

### Example:

```JS
button: {
		backgroundColor: "#123456",
                right: 40,
                bottom: 25,
                size: 80,
		iconColor: "#654321",
                customIconSrc: "../icon.svg",
            },
```

**Reply by amansoni7477030** [reply, marked as answer]
Hii @cpapxl I have added the functionality for this, and in addition, I have also implemented drag and drop functionality.
Check https://github.com/FlowiseAI/FlowiseChatEmbed/blob/main/README.md for configuration.
